From c7b72ce809b160f239f8ccab1de8bdb8d9e50508 Mon Sep 17 00:00:00 2001
From: Ahmed Mansour <Ahmed.Mansour@nxp.com>
Date: Thu, 26 May 2016 17:17:45 -0400
Subject: [PATCH 522/666] fsl_qbman/qman_high.c: Remove debug print

Removed debug print left in source by accident

JIRA issue#: QLINUX-5727

Signed-off-by: Ahmed Mansour <Ahmed.Mansour@nxp.com>
[Original patch taken from SDK-V2.0-1703]
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/staging/fsl_qbman/qman_high.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/staging/fsl_qbman/qman_high.c b/drivers/staging/fsl_qbman/qman_high.c
index 5ec9db3..228b7e6 100644
--- a/drivers/staging/fsl_qbman/qman_high.c
+++ b/drivers/staging/fsl_qbman/qman_high.c
@@ -1091,7 +1091,6 @@ static u32 __poll_portal_slow(struct qman_portal *p, u32 is)
 		} else {
 			/* Its a software ERN */
 #ifdef CONFIG_FSL_QMAN_FQ_LOOKUP
-			pr_info("ROY\n");
 			fq = get_fq_table_entry(be32_to_cpu(msg->ern.tag));
 #else
 			fq = (void *)(uintptr_t)be32_to_cpu(msg->ern.tag);
-- 
2.7.4

